src/v2/Apps/Home/Components/HomeFeaturedMarketNews.tsx

```diff
@@ -136,9 +147,11 @@ const HomeFeaturedMarketNews: React.FC<HomeFeaturedMarketNewsProps> = ({
                     <Text variant="xs" textTransform="uppercase" my={1}>
                       {article.vertical}
                     </Text>
-
-                    <Text variant="lg">{article.title}</Text>
-
+                    
+                    <StyledTitle to={article.href ?? ""}>
+                      <Text variant="lg">{article.title}</Text>
+                    </StyledTitle>
+                    
```

Comment:
Ditto, just style the parent one and then if you want you can target specific sub-nodes.

https://styled-components.com/docs/advanced#referring-to-other-components

Reply:
Incorporated the `RouterLink`